en/deploy-failures.md

```diff
@@ -130,6 +130,44 @@ root soft core unlimited
 root soft stack 10240
 ```
 
+### PD Pod nslookup domain failed
+
+You should see some log of PD Pod like:
+
+```
+Thu Jan 13 14:55:52 IST 2022
+;; Got recursion not available from 10.43.0.10, trying next server
+;; Got recursion not available from 10.43.0.10, trying next server
+;; Got recursion not available from 10.43.0.10, trying next server
+Server: 10.43.0.10
+Address: 10.43.0.10#53
+
+** server can't find basic-pd-0.basic-pd-peer.default.svc: NXDOMAIN
+
+nslookup domain basic-pd-0.basic-pd-peer.default.svc failed
+```
+
+This happen when:
+
+- There two `nameserver` in `/etc/resolv.conf` and the second one is not IP of CoreDNS.
+- Version of PD is:
+    - Greater than or equals v5.0.5.
+    - Greater than or equals v5.1.4.
+    - Greater than or equals v5.2.4.
+    - All 5.3 versions.
+
+To address this, add `startUpScriptVersion` to TiDBCluster as:
+
+```yaml
+...
+spec:
+  pd:
+    startUpScriptVersion: "v1"
+...
+```
+
+The reason is nslookup version is changed in the image, See: [#4379](https://github.com/pingcap/tidb-operator/pull/4379). after adding `startUpScriptVersion` as `v1`, we will use `dig` to check DNS instead of using nslookup.
```

Comment:
```suggestion
The reason is that there is something wrong with the nslookup in the base image, see detail in the description of [#4379](https://github.com/pingcap/tidb-operator/pull/4379). After adding `startUpScriptVersion` as `v1`, we will use `dig` to check DNS instead of using nslookup.
```